lib/commands/web.js

```diff
@@ -433,12 +437,20 @@ helpers.parseExecuteResponse = function (res) {
 };
 
 helpers.parseElementResponse = function (element) {
-  let objId = element.ELEMENT;
+  let objId = helpers.getElementId(element);
   let clientId = (ELEMENT_OFFSET + this.webElementIds.length).toString();
   this.webElementIds.push(objId);
   return {ELEMENT: clientId};
 };
 
+helpers.getElementId = function (element) {
+  return element.ELEMENT || element[W3C_WEB_ELEMENT_IDENTIFIER];
+};
+
+helpers.hasElementId = function (element) {
+  return util.hasValue(element) && (util.hasValue(element.ELEMENT) || util.hasValue(element[W3C_WEB_ELEMENT_IDENTIFIER]));
```

Comment:
and this can be also changed to `return util.unwrapElement(element) !== element`